src/Dashboard/Resources/Resources.tsx

```diff
@@ -28,7 +28,9 @@ const defaultResources: Resource[] = [
         name: "Curriculum",
       },
     ],
-    tags: ["Unit 2: AI & You", "Worksheet", "Video"],
+    unitTags: ["Unit 2: AI & You"],
+    typeTags: ["Worksheet", "Video"],
+    miscTags: []
   },
   {
     title: "“What the world looks like to an algorithm” YouTube Video",
```

Comment:
I'm a little worried that we're going to have to rewrite all this code if we want to add additional filters. Danielle, for example, was wondering if we could add a filter for `age` or for the `materials` that students might need. Ideally, we'd have this as

```js
tags: {
    "unit": ["Unit 2: AI & You"],
    "type": ["Worksheet", "Video"]
}
```

and then the logic above would use the `keys` of the `tags` variable, instead of hardcoding unit and type.